projects.yaml

```diff
@@ -27,4 +27,24 @@ PAM_MUJOCO:
 
 BALL_TRACKING:
     parent_projects: ['AMENT']
-    repos: ['tennicam_proto','tennicam','mpi_cmake_modules']
\ No newline at end of file
+    repos: ['tennicam_proto','tennicam','mpi_cmake_modules']
```

Comment:
I would remove tennicam and tennicam_proto, they are a superficial refactor of legacy code that would take possibly too much time to make proper

Reply:
my bad, went too fast, all good